### DISCUSSION TITLE
Partial Selector matching.

### DISCUSSION THREAD

**Discussion by taruntechno** [seleniumbase/SeleniumBase#1213] [Q&A] [answered]
Hello @mdmintz does sb have support for partial matching of selectors.

For eg angular code is genrating id's and name  as = _testId_y12ht. So only the testId part stays consistent so can I match partially on that.
For eg self.click("input#_testId_y12ht") works for a single session but as soon as page refresh it would fail.

**Comment by mdmintz** [comment, marked as answer]
@taruntechno You can use ``'input[id*="_testId_"]'`` for that selector for a partial ID match.
